kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -222,9 +223,23 @@ export class FormWizard extends HTMLElement {
       let activeStepIndex = this.#steps.findIndex(
         ({ name }) => name === this.activeStep
       );
+      let activeStepCount = activeStepIndex + 1;
       let progress =
         Math.ceil((activeStepIndex / (this.#steps.length - 1)) * 100) || 10;
-      this.#progressIndicator.value = progress;
+      let indicator = this.#progressBar.querySelector(".indicator");
+      indicator.style.setProperty("--progress", progress + "%");
+      // Ensure progressbar starts translated at -100% before we
+      // set --progress to avoid a layout race.
+      requestAnimationFrame(() => {
+        requestAnimationFrame(() => {
+          indicator.toggleAttribute("ready", true);
+        });
+      });
+      this.#progressBar.setAttribute("aria-valuenow", activeStepCount);
+      this.#progressBar.setAttribute(
+        "aria-label",
+        interpolate(gettext("Step %s of 3"), [activeStepCount])
+      );
```

Comment:
Seems like we're hard-coding a total of `3` both here and in the markup. Since the form-wizard is meant to be a bit more general, I think we can probably write that value in based on `this.#steps.length - 1`.

Reply:
good catch, thank you!